src/screens/Sidebar/SidebarScreen.js

```diff
@@ -71,21 +72,25 @@ class SidebarScreen extends React.Component {
         key: 'groups',
         data: groupsData,
         title: 'Groups',
-        renderItem: ({ item }) => (
-          <SidebarListItem
-            onPress={() => {
-              navigation.navigate('');
-            }}
-            title={item.group.name}
-            imageURL={
-              item.group.avatar.small ||
-                item.group.avatar.medium ||
-                item.group.avatar.large ||
-                item.group.avatar.original ||
-                defaultAvatar
-            }
-          />
-        ),
+        renderItem: ({ item }) => {
+          console.log(item);
+          return (
+            <SidebarListItem
+              onPress={() => {
+                navigation.navigate('');
+              }}
+              title={item.group.name}
+              imageURL={
+                (item.group.avatar &&
+                  (item.group.avatar.small ||
```

Comment:
we should write a helper for this I think -- not anything you need to do but I'm just thinking in text :D